.prettierrc.json

```diff
@@ -1 +1,6 @@
-{}
+{
+  "printWidth": 120,
+  "singleQuote": true,
```

Comment:
🔶 Why change the default on this one? I think Prettier make a good case for double quote default here: https://prettier.io/docs/en/rationale.html#strings

Reply:
Fair argument. I do find single quote "less visually noisy" which is why I prefer it. But that's preference, I can't make objective argument for it which is why I won't defend. Should we see preferences and consensus from others?

Reply:
Perhaps a custom IDE font would help with visual noise?

Reply:
I would hope to keep things as default as much as possible. Concerning personal tastes/likings this is something we can adjust along the way with whoever will use it. We first set a baseline. We iterate after on it (if needed).